api/app/views/mno_enterprise/jpi/v1/marketplace/app_reviews.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+json.number_reviews @reviews_number
```

Comment:
No need for this.
@reviews_number is not defined.
We already send the total count in the headers